defaults/main.yml

```
@@ -1,4 +1,5 @@
 ---
 
-# List of files to be found under `potos_wallpaper` to be added as background, first element is set as default
+# List of files to be found under `potos_wallpaper` to be added as background,
+# first element is set as default
 potos_wallpaper_images = []
```

Needed to fix the pipeline

```suggestion
potos_wallpaper_images: []
```